### PR TITLE
[Sprint 1 - Task 6] - Implement script to create Application Gateway Account

### LINKED ISSUE #28: [Task] Task 6 - Implement script to create Application Gateway account
Write a script to create Application Gateway Account by using Azure CLI PowerShell.

### PR DESCRIPTION
**Pull request by himawaritane**
Complete the script of implement Application Gateway Account.

### CODE REVIEW

**Review by RayWu222 — DISMISSED** [review]
I am getting errors trying to run this code. The errors come from Madatory and not Mandatory. I am also getting an error in the resource group name after changing the word to mandatory.

**Comment by sunny852206** [comment]
> I am getting errors trying to run this code. The errors come from Madatory and not Mandatory. I am also getting an error in the resource group name after changing the word to mandatory.

I am getting the same errors when running the code as well.

**Review by himawaritane — COMMENTED** [review]
Change the word "Madatory" to "Mandatory" on script file.

**Comment by himawaritane** [comment]
> I am getting errors trying to run this code. The errors come from Madatory and not Mandatory. I am also getting an error in the resource group name after changing the word to mandatory.

I changed the word, can you run the code one more time?

**Comment by himawaritane** [comment]
> > I am getting errors trying to run this code. The errors come from Madatory and not Mandatory. I am also getting an error in the resource group name after changing the word to mandatory.
> 
> I am getting the same errors when running the code as well.

I fixed the error.

File: exmple/deploy-AppGateway.ps1 [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    [string]
+    $SubcriptionID,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $tenantID,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $ResourGroupName,
```
Comment (RayWu222) [inline exmple/deploy-AppGateway.ps1]: Please change to ResourceGroupName. This is causing an error

**Comment by RayWu222** [comment]
I reviewed the recent commit. I am still getting an error, this time is from the resource group name. I put a comment on the line that needs to be changed.

File: exmple/deploy-AppGateway.ps1 (hunk 1 above)
Comment (himawaritane) [reply exmple/deploy-AppGateway.ps1]: I got it.

**Review by jlsnow301 — APPROVED** [review]

**Comment by jlsnow301** [comment]
Looks like the requested changes were made / I don't see any errors, merging.

**Pull request merged**